### DISCUSSION TITLE
Site replication fails with Error: unable to create local service account:

### DISCUSSION THREAD

**Discussion by woss** [minio/minio#16980] [Q&A] [answered]
Hi,

i'm trying to replicate my minio setup to another machine using the same settings ( my infra docker-compose files ) which are the ones you provide here https://github.com/minio/minio/blob/master/docs/orchestration/docker-compose/docker-compose.yaml

The ports are open and the containers can talk to each other. What could be the cause of this error?

This is the role for the API key on both sides
```json
{
 "Version": "2012-10-17",
 "Statement": [
  {
   "Effect": "Allow",
   "Action": [
    "admin:*"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "kms:*"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "s3:*"
   ],
   "Resource": [
    "arn:aws:s3:::*"
   ]
  }
 ]
}
```

![image](https://user-images.githubusercontent.com/80440/230161574-d851cd82-a570-4b1a-8f75-8fadec91b43c.png)

Thanks

**Comment by harshavardhana** [comment]
@poornas ^^

**Comment by clabeuhtegrite** [comment]
I had the same problem. Trying to add a replication site result in the following error: 

Site replication related service account error (unable to create local service account: Specified service account action is not allowed: unable to create a service account for another service account)

**Reply by ghost** [reply]
I have the same problem, and not sure how to proceed. can anyone share any information on what we can do to resolve this?

**Reply by poornas** [reply, marked as answer]
the aliases used in `mc admin replicate add` should be that of admin user. Looks like you are using a service acct as alias?

**Reply by hieuttbk** [reply]
how i can fix it to done with UI?

**Comment by jaredmcqueen** [comment]
I had this problem too - turns out I didn't have to use keys, and instead used the `root` user's username and password

**Reply by jmbarreiro1** [reply]
I also have a similar issue

**Reply by mehdiMj-ir** [reply]
I had the same issue and it fixed using `minioadmin`'s username and password but is this best practice?

**Comment by mahoor13** [comment]
I used minio root password for both sides to fix this issue.
